src/symbols.js

```diff
@@ -240,16 +241,16 @@ defineSymbol(math, ams, rel, "\u22e9", "\\succnsim", true);
 defineSymbol(math, ams, rel, "\u2aba", "\\succnapprox", true);
 // unicode-math maps \u2246 to \simneqq. We'll use the AMS synonym.
 defineSymbol(math, ams, rel, "\u2246", "\\ncong", true);
-defineSymbol(math, ams, rel, "\ue007", "\\nshortparallel");
+defineSymbol(math, ams, rel, "\ue007", "\\@nshortparallel");
 defineSymbol(math, ams, rel, "\u2226", "\\nparallel", true);
 defineSymbol(math, ams, rel, "\u22af", "\\nVDash", true);
 defineSymbol(math, ams, rel, "\u22eb", "\\ntriangleright");
 defineSymbol(math, ams, rel, "\u22ed", "\\ntrianglerighteq", true);
 defineSymbol(math, ams, rel, "\ue018", "\\nsupseteqq");
 defineSymbol(math, ams, rel, "\u228b", "\\supsetneq", true);
-defineSymbol(math, ams, rel, "\ue01b", "\\varsupsetneq");
+defineSymbol(math, ams, rel, "\ue01b", "\\@varsupsetneq");
 defineSymbol(math, ams, rel, "\u2acc", "\\supsetneqq", true);
-defineSymbol(math, ams, rel, "\ue019", "\\varsupsetneqq");
+defineSymbol(math, ams, rel, "\ue019", "\\@varsupsetneqq");
 defineSymbol(math, ams, rel, "\u22ae", "\\nVdash", true);
 defineSymbol(math, ams, rel, "\u2ab5", "\\precneqq", true);
 defineSymbol(math, ams, rel, "\u2ab6", "\\succneqq", true);
```

Comment:
Maybe `u2289`? (next line)

Reply:
Well, those are interesting suggestions. If I understand matters correctly, I am proposing Option A below and you are proposing Option B

| Function     | LaTeX | Option A | Option B |
|:------------|:--------:|:----------:|:----------:|
| \varsubsetneq |  ![varsubsetneq](https://user-images.githubusercontent.com/16403058/54962697-0d791700-4f23-11e9-89e3-e8b829478113.png)     |  ⊊         |    ⊈   |
| \varsupsetneq |  ![varsupsetneq](https://user-images.githubusercontent.com/16403058/54962637-d0ad2000-4f22-11e9-8d13-c19c15f8ac15.png)     |  ⊋         |    ⊉   |

I find the LaTeX symbols to be ambiguous and annoying. Either Option A or Option B is less ambiguous but I am not completely sure what the LaTeX symbols are trying to convey. If I had to bet, I'd pick Option A. For what it's worth MathMax and TeXZilla made the same choice that I made, as you can see [here](http://webmathcomparison.net/#v).

BTW, I've updated that page with `\middle`. Thank you for the quick review.

Reply:
The name "subsetneq" suggests "subset but not equal" which seems to be Option A, while Option B means "not subset". Presumably the Unicode character names confirm this?

Reply:
Unicode character names for Option A are: "Subset of with Not Equal To" and "Superset of with Not Equal To", which I think is more corroboration.

Reply:
@ronkok I meant for the `\nsupseteqq` and `\nsubseteqq`, which also use PUAs.

Reply:
@ylemkimon Thank you for the clarification. I have revised this PR to include `\nsubseteqq` and `\nsupseteqq`.